Kernel/API/InodeWatcherEvent.h

```diff
@@ -9,13 +9,18 @@
 #include <AK/Types.h>
 
 struct [[gnu::packed]] InodeWatcherEvent {
-    enum class Type {
+    enum Type {
```

Comment:
```suggestion
    enum class Type {
```

Was this changed intentionally?

Reply:
Yes. This makes it easier to use it as flags in `event_flags` when setting up watchers and when checking against existing watchers in `notify_inode_event` (otherwise you have to `static_cast` everywhere).

Reply:
Hm yeah, but then you pollute the global namespace with names like `Invalid` and `Deleted`.

Reply:
Could you check if AK/EnumBits can help out here?
It will add bitwise ops and a `has_flag(T, T)` function

Reply:
Yea EnumBits was added for this purpose.

Reply:
I've tried that, however now the syscalls require manual `static_cast`ing, and making the event flags argument an enum class isn't possible because C code uses those headers.